scripts/core/auth/auth.js

```diff
@@ -90,6 +90,26 @@ function ResetPassworController($scope, $location, api, notify, gettext) {
     }
 }
 
+SecureLoginController.$inject = ['$scope', 'auth', '$route', '$window'];
```

Comment:
Let's document this too. An example can be seen [here](https://github.com/superdesk/superdesk-client-core/blob/master/scripts/apps/authoring/suggest/SuggestController.js).